### PR TITLE
a few billing fixes

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #

### CODE REVIEW

File: library/patient.inc.php [hunk 1]
```diff
@@ -1716,7 +1718,9 @@ function get_patient_balance($pid, $with_insurance = false, $eid = false)
                 "deleted IS NULL AND pid = ? AND encounter = ? AND payer_type = 0",
                 array($pid, $encounter)
             );
-            $copay = !empty($insarr[0]['copay']) ? $insarr[0]['copay'] * 1 : 0;
+            // going to comment this out for now since computing future copays doesn't
+            // equate to cash in hand, which shows in the Billing widget in dashboard 4-23-21
+            // $copay = !empty($insarr[0]['copay']) ? $insarr[0]['copay'] * 1 : 0;
```
Comment (stephenwaite) [inline library/patient.inc.php:1724]: for this see these images to understand how a encounter that didn't even have a fee sheet was incorrectly reflecting cash based on the copay entered in demographics
before:
Screenshot from 2021-04-23 16-35-29
after:
Screenshot from 2021-04-23 16-56-31
demos:
Screenshot from 2021-04-23 16-57-39

File: interface/forms/eye_mag/report.php [hunk 2]
```diff
@@ -33,7 +33,7 @@
 require_once(dirname(__FILE__) . "/../../../library/api.inc");
 require_once(dirname(__FILE__) . "/../../../library/lists.inc");
 require_once(dirname(__FILE__) . "/../../../library/forms.inc");
-require_once(dirname(__FILE__) . "/../../../library/patient.inc");
+require_once(dirname(__FILE__) . "/../../../library/patient.inc.php");
```
Comment (bradymiller) [inline interface/forms/eye_mag/report.php:36]: is the .inc not working? (note there are quite a few .inc scripts included above here)

**Comment by bradymiller** [comment]
tenor99

**Comment by stephenwaite** [comment]
not a big deal about the patient.inc, just can't use xdebug in vscode with that suffix

**Comment by bradymiller** [comment]
that is annoying. will that break with all *.inc scripts. we could just convert them all from *.inc to *.php instead.

**Comment by bradymiller** [comment]
oh goodness, just checked. that would be quite the project to convert from .inc to .php and then would also probably break folks with custom code.

**Comment by sunsetsystems** [comment]
Would it help to do the renames, then create new .inc files that just "require" the corresponding .inc.php files?

**Comment by bradymiller** [comment]
That would work and would then not break custom code (and we could remove those .inc files at some defined point in the future).

**Comment by sunsetsystems** [comment]
I'd just also suggest adding the new .inc files in a separate commit and not squashing, so git will remember that the renames happened.

**Comment by sjpadgett** [comment]
This will touch scores of files! What is gained with this mass refactor? Also consider what's going to happen when we want to patch released version of v6. This could be something to do just before a major release where v6 is young and we'll see several patches before ONC release.

**Comment by stephenwaite** [comment]
looking at a lot of the functions in `patient.inc`, maybe they could be refactored into `src/Services`?

**Comment by stephenwaite** [comment]
will take up the `.inc` refactor in the future maybe

**Review by bradymiller — APPROVED** [review]

**Pull request merged**